### PR TITLE
The payment history article needs a section about invoice states

### PR DESCRIPTION
**Pull request by jacegu**
As discussed on #78

### CODE REVIEW

**Comment by jacegu** [comment]
I think this is a good starting point. We can add or remove as required in the future.

Note that I've intentionally left out the `written_off` state.

**Comment by jacegu** [comment]
Ready for your :eyes: @aetrion/dnsimple

File: content/articles/account-payment-history.markdown [hunk 1]
```diff
@@ -28,6 +28,47 @@ It is possible to view your payment history and retry failed invoices manually i
...
+
+When you make a purchase like a domain registration or an SSL certificate and we are not able to collect the payment for it you will be notified via email and the invoice will be marked as `failed`.
+
+While the invoice is in this state you won't be able to make other purchases. Note that you will be able to [retry the payment](#retrying) yourself and our systems will also try to collect it every few hours.
+
+### `dunned` {#invoice-state-dunned}
+
+Depending of your plan your subscription will be renewed every month or every year. When that happens we attempt to collect the payment and if it fails the invoice will be marked as `dunned`.
+
+A `dunned` invoice will be retried every day and you will get notified via email on every attempt. If we are unable to collect the payment after a certain period of time your account will be cancelled.
```
Comment (weppos) [inline content/articles/account-payment-history.markdown]: I'd highlight this section in a `<warning>`.

```
<warning>
If we are unable to collect the payment after a certain period of time your account will be cancelled and all the domains within the account will stop resolving immediately.
</warning>
```

**Comment by weppos** [comment]
It looks good to me! :+1:

**Comment by jacegu** [comment]
Warning section added. Thanks for the review @weppos.

I'll wait for @jcaudle to take a look and merge it.

**Comment by jcaudle** [comment]
This looks good to me too.

**Pull request merged**